.github/RELEASE_SCHEDULE.md

```diff
@@ -1,16 +1,29 @@
 
-The goal of these release candidates is to give the community time to adjust their existing environments for any breaking changes, as well as to identify any issues with the changes.  If necessary, changes will be incorporated an additional RC release could be made if significant problems are identified.  If a revised Release Candidate is necessary the Production Release schedule may be impacted.  The exact impact will vary on a case-by-case basis depending on the nature of the issue(s) identified during RC review, however, will be clearly communicated as part off the updated RC release notes.
+### Major Releases (Example 10.0.0)
+Major releases will have an initial RC cycle with a minimum duration of three-weeks from the date of RC release and final release. 
+
+### Minor Releases (Example 9.8.x, or 9.9.x)
+Minor releases will have an initial RC cycle with a minimum duration of two weeks from the date of RC release and final release.
+
+### Revision Releases (Example 9.8.1, or 9.8.2)
+Revision releases will have an initial RC cycle with a minimum duration of one week from the RC release and final release.
+
+### Changes During RC Phase
+If necessary, changes will be incorporated during an RC; if the changes resolve significant issues or introduce risk, an additional RC may be created at the Approvers group's discretion.  If a revised RC is necessary, the Production Release schedule will be impacted; the exact impact will vary on a case-by-case basis depending on the nature of the issue(s) identified during the RC review. However, it will be communicated as part of the updated RC release notes.
 
 ## Production Releases
-Production releases will only be completed after a successful RC phase, except in the case of a significant security release that was included as part of a revision release.
+Production releases will only be completed after a successful RC phase, except in the case of a significant security release included as part of a revision release.
 
 The anticipated release date will be communicated to the community at the time of the RC.  And each release will take the following considerations into mind for all releases.
 
-* Releases must allow for at least two additional business days after the release for regular Monday - Friday office situations (Releases only on Monday, Tuesday or Wednesday)
+* Releases must allow for at least two additional business days after the release for regular Monday - Friday office situations (Releases only on Monday, Tuesday, or Wednesday)
 * Releases will not be completed during weeks of major US holidays, specifically New Years, Memorial Day, Independence Day, Labor Day, Thanksgiving Day, or Christmas.
 * Best efforts shall be made to avoid other significant holidays in other countries.
+
+## Release Notifications
+You can utilize the "Watch" functionality within GitHub to receive notifications for new RC & Production releases using the "Releases Only" notification option.
```

Comment:
```suggestion
You can utilize the "Watch" functionality within GitHub to receive notifications for new Release Candidates and/or Official Releases using the "Releases Only" notification option.
```

Reply:
It think it needs to be `and` rather than `and/or`, since you can't just subscribe to "official" releases.

Reply:
Thanks - I updated.